Library/ViewModels/ChangeEmailViewModel.swift

```diff
@@ -207,14 +200,6 @@ ChangeEmailViewModelOutputs {
     self.viewDidLoadProperty.value = ()
   }
 
-  private let changePasswordProperty = MutableProperty<(String, String)?>(nil)
-  public func submitForm(newEmail: String?, password: String?) {
-
-    if let newEmail = newEmail, let password = password {
-      self.changePasswordProperty.value = (newEmail, password)
-    }
-  }
-
   private let saveButtonTappedProperty = MutableProperty<(String, String)?>(nil)
   public func saveButtonTapped(newEmail: String?, password: String?) {
```

Comment:
We shouldn't have any logic in the input function itself, it should be handled via signals.

Also - I don't think these values should be passed in this way, we're setting them on every keypress so we should be able to just have combine those signals and grab the values on our save button press using `takeWhen`.